content/_img-sources/ticket-creation.uxf

```diff
@@ -0,0 +1,208 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
These `.uxf` files are the UMLet source files for the diagrams. You don't have to review them directly (they are exported as `.svg` files that are also committed to the repo)